azerothcore/azerothcore-wotlk
# fix(Scripts/UBRS): Warmaster Voone stances

* Add missing Thresh ability
* Change abilities according to combat stances



## Changes Proposed:
- Add missing Thresh ability
- Change abilities according to combat stances
- He now casts Throw Axe on a random target, as suggested by his wowwiki page

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7362

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. Go to Warmaster Voone
2. Check that he changes his set of abilities after damaged (65, 40%)
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
